### PR TITLE
Added webhook and message template support to zabbix_mediatype

### LINKED ISSUE #79: mediatype now supports new webhook type since 4.4
##### SUMMARY
Webhook since 4.4:
https://www.zabbix.com/documentation/4.4/manual/api/reference/mediatype/object

Message template since 5.0 (even tho it looks more like error in documentation - will take a closer look at this):
https://www.zabbix.com/documentation/current/manual/api/reference/mediatype/object

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
plugins/modules/zabbix_mediatype

### PR DESCRIPTION
**Pull request by D3DeFi**
##### SUMMARY
New features to `zabbix_mediatype` module:
- new type webhook since 4.4
- description option supported since 4.4
- message templates can be provided since 5.0
- some code optimalizations to `construct_parameters()` as there was too much code repetition
- tests for the new features added

Fixes #79 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/modules/zabbix_mediatype.py

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

**Comment by D3DeFi** [comment]
@sky-joker can you look at this please?

**Comment by sky-joker** [comment]
Sorry, I’ll check this PR in a few days.

File: tests/integration/targets/test_zabbix_mediatype/tasks/main.yml [hunk 1]
```diff
@@ -150,6 +150,115 @@
...
+    - name: test - reset email mediatype smtp information to default
+      zabbix_mediatype:
+      register: zbxmediatype_reset
+
+    - assert:
+        that: zbxmediatype_reset.changed is sameas True
+
+    - name: test - update email mediatype with message templates
+      zabbix_mediatype:
+        message_templates:
```
Comment (sky-joker) [inline tests/integration/targets/test_zabbix_mediatype/tasks/main.yml:164]: Can you please add `eventsource: internal` ?

Comment (D3DeFi) [reply tests/integration/targets/test_zabbix_mediatype/tasks/main.yml:164]: done

**Comment by sky-joker** [comment]
Thank you @D3DeFi  for this patch :)
LGTM

**Comment by D3DeFi** [comment]
> LGTM

Thank you for your time and looking into this

**Pull request merged**